db_express_api/src/routes/maintenance.js

```diff
@@ -46,11 +62,11 @@ router.get('/maintenance/:id', function (req, res) {
 
 // Plan maintenance
 router.post('/', function (req, res) {
-    var params = req.body;
-    params.is_complete = "0";
-    params.is_canceled = "0";
+    var params = req.body.params;
```

Comment:
Might want to check the params here to make sure they are correct. Might be good to also do that in the UI. Just for defensive coding.